onionbalance/hs_v3/service.py

```diff
@@ -263,6 +263,28 @@ def publish_descriptors(self):
         self._publish_descriptor(is_first_desc=True)
         self._publish_descriptor(is_first_desc=False)
 
+    
+    def _get_intros_for_distinct_desc(self, maxCount=10):
```

Comment:
Nitpick: `maxCount` is camel cased, when the rest of the code is underscore-based. Let's make all new code use underscores please.